### ISSUE TITLE
Web server not responding

### ISSUE THREAD

**Issue by ryandesign** [pinard/Recode#8]
Your homepage is inaccessible; the web server is not responding

```
$ curl http://recode.progiciels-bpi.ca/
curl: (7) Failed to connect to recode.progiciels-bpi.ca port 80: Connection refused
```

**Comment by TTimo** [comment]
+1 site not responding

**Comment by rrthomas** [comment]
The site is dead. I have taken over the project at https://github.com/rrthomas/

**Comment by ryandesign** [comment]
https://www.gnu.org/software/recode/ still links to the old (dead) site.

**Comment by rrthomas** [comment]
Recode hasn't been a GNU program for some years, but I'll try to get the GNU webmasters to remove it, because it's not helpful. Thanks!

**Comment by rrthomas** [comment]
(Also, not much point filing issues here, I can't control this project.)

**Comment by ryandesign** [comment]
Hopefully GNU will update the link, rather than remove the page. The project was known by that URL at one time, and many package management systems still link there; see its page on [repology](https://repology.org/metapackage/recode/information). Updating the link there may help inform package maintainers that your fork exists.

**Comment by rrthomas** [comment]
You're quite right, and I came to the same conclusion while emailing the GNU webmasters.

**Comment by rrthomas** [comment]
Done!
